runtime/src/accounts_db.rs

```diff
@@ -7233,17 +7093,12 @@ pub mod tests {
             Hash::from_str("4StuvYHFd7xuShVXB94uHHvpqGMCaacdZnYB74QQkPA1").unwrap();
 
         assert_eq!(
-            AccountsDb::hash_stored_account(slot, &stored_account, &ClusterType::Development),
+            AccountsDb::hash_stored_account(slot, &stored_account),
             expected_account_hash,
             "StoredAccountMeta's data layout might be changed; update hashing if needed."
         );
         assert_eq!(
-            AccountsDb::hash_account(
-                slot,
-                &account,
-                &stored_account.meta.pubkey,
-                &ClusterType::Development
-            ),
+            AccountsDb::hash_account(slot, &account, &stored_account.meta.pubkey,),
```

Comment:
nits: you can remove the `,` immediately before `)`.

Reply:
Done!